tox.ini

```diff
@@ -13,17 +13,23 @@ envlist =
     test-py37-django{111,20,22}-drf310-jsonapi28-yasg170
     test-py37-django{22,30}-drf311-jsonapi{31,32}-yasg171
     # yasg200 and jsonapi40 require drf312 or higher
-    test-py37-django{22,30,31}-drf312-jsonapi40-yasg200
+    test-py37-django{22,30,31}-drf312-jsonapi{40,41,42,43}-yasg200
```

Comment:
Can you replace it with `jsonapi{40,43}`?  – I assume testing first release of new major version (`4.0`) and latest  (`4.3`) is just enough in our case

And BTW since we now explicitly support 3.2 (which is current LTS) can replace `django{22,30,31}` with `django{22,30,32}`? – again I want to test against first release of new major version (`3.0`) and LTS release

(Same for some python 3.8 and 3.9 below)

Reply:
Sure, made the required changes. Had to keep `django32` in separate lines though as there is no compatibility with previous versions of `jsonapi`

Reply:
:+1: ok, makes sense